inference-engine/src/mkldnn_plugin/nodes/mkldnn_multiclass_nms.cpp

```diff
@@ -0,0 +1,566 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "base.hpp"
+
+#include <algorithm>
+#include <cassert>
+#include <cmath>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph_ops/nms_static_shape_ie.hpp>
+#include <queue>
+#include <string>
+#include <utility>
+#include <vector>
+#include "ie_parallel.hpp"
+#include "mkldnn_multiclass_nms.hpp"
+#include "utils/general_utils.h"
+
+#include <chrono>
+
+using namespace MKLDNNPlugin;
+using namespace InferenceEngine;
+
+bool MKLDNNMultiClassNmsNode::isSupportedOperation(
+    const std::shared_ptr<ngraph::Node>& op,
```

Comment:
max line length is 160 so fix alignment please

Reply:
alignment will be set to 160 in next commit(align with 'inference-engine/samples/.clang-format').